tests/AUTOMATE_TEST_GUIDE.md

```diff
@@ -2,7 +2,19 @@
 
 This style guide provides editorial guidelines for anyone creating new automated test cases for the CHT-Core.
 
-There are three files that are the base of every new automated test case, the most important one is the `<name>.wdio.spec.js`, which contains the actual test that will be executed. It needs to have that name convention because the [config file](https://github.com/medic/cht-core/blob/master/tests/wdio.conf.js#L42-L44) has as an attribute to execute all the files that have `*.wdio-spec.js`.
+There are three files that are the base of every new automated test case, the most important one is the `spec.js`, which contains the actual test that will be executed. 
+
+Right now we have two different configs set up:
+* **default**
+    * Config file: [`wdio.conf.js`](https://github.com/medic/cht-core/blob/master/tests/wdio-standard.conf.js)
+    * Name convention for the `spec` file: `<name>.wdio-spec.js`
+    * Command to execute the tests that belong to this config:  `npm run wdio-local`
+* **standard**
+    * Config file: [`wdio-standard.conf.js`](https://github.com/medic/cht-core/blob/master/tests/wdio.conf.js)
+    * Name convention for the `spec` file: `<name>.standard-wdio-spec.js`
```

Comment:
> Create the standard folder inside e2e. I removed the standard part from the name of the js files because as you said, they are already in the standard folder. This is what it contains

This is great! We'll have 1 pattern, easy to understand, easy to replicate if we automate more configs like BRAC. Thank you!

> Update the command to execute the standard test: npm run standard-wdio-local because it makes more sense that the word standard is at the beginning

Sounds good 👍